HealthSDK/GiniHealthSDKPinning/Package-release.swift

```diff
@@ -0,0 +1,31 @@
+// swift-tools-version:5.5
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "GiniHealthSDKPinning",
+    platforms: [.iOS(.v12)],
+    products: [
+        // Products define the executables and libraries a package produces, and make them visible to other packages.
+        .library(
+            name: "GiniHealthSDKPinning",
+            targets: ["GiniHealthSDKPinning"]),
+    ],
+    dependencies: [
+        // Dependencies declare other packages that this package depends on.
+        // .package(url: /* package url */, from: "1.0.0"),
+        .package(name: "GiniHealthAPILibraryPinning", url: "https://github.com/gini/health-api-library-pinning-ios.git", from: "0.0.1"),
+        .package(name: "GiniHealthSDK", url: "https://github.com/gini/health-sdk-ios.git", from: "0.0.1"),
```

Comment:
Here we should also use exact versions to have deterministic version resolution for our packages.